Resources/Private/Partials/Page/Nav/SkipLinks.html

```diff
@@ -1,6 +1,6 @@
 <html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true">
 
-<div class="skip-link__wrp">
+<div class="skip-link__wrp" role="navigation" aria-label="{f:translate(key: 'skip_to_main_content', extensionName: 't3kit')}">
```

Comment:
I am not sure about `role="navigation"` it is just a link.
Why do we need aria-label with the same text as a link text?
Maybe I am missing something, let's discuss.